catkin_home/src/devices/CMakeLists.txt

```diff
@@ -74,6 +74,7 @@ find_package(catkin REQUIRED COMPONENTS
    DEPENDENCIES
    std_msgs  # Or other packages containing msgs
    audio_common_msgs
+   intercom
```

Comment:
Is it necessary to include `intercom` in the `find_package()`?

Reply:
it is! , I removed previously the connection with main_engine and forgot about it, thanks for pointing it out.